db/migrations/20210324093804_create_dog_file_vow.sql

```diff
@@ -5,7 +5,7 @@ CREATE TABLE maker.dog_file_vow
     log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
     address_id BIGINT  NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
     header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
-    what       BIGINT,
+    what       TEXT,
```

Comment:
Has this already been applied in QA? If so, you probably need a migration to change it. If there's no data in there, you should be able to modify the column.

Reply:
I don't think there was ever data in there since the test wasn't working